maths/prime_numbers.py

```diff
@@ -1,7 +1,7 @@
-from typing import List
+from typing import Generator, List
 
 
-def primes(max: int) -> List[int]:
+def primes(max: int) -> Generator[int, None, None]:
     """
     Return a list of all primes numbers up to max.
     >>> primes(10)
```

Comment:
Please change this to __list(primes(10))__ so this doctest passes.

Reply:
fixed